src/main/kotlin/com/terraformation/backend/tracking/db/ObservationResultsStore.kt

```diff
@@ -280,6 +282,7 @@ class ObservationResultsStore(private val dslContext: DSLContext) {
                         TREE_GROWTH_FORM_ID,
                         TREE_NUMBER,
                         TRUNK_NUMBER,
+                        recordedTreesGpsCoordinatesField,
```

Comment:
What's the benefit here of the separate variable as opposed to just doing `GPS_COORDINATES`?

Reply:
It's a workaround for a bug/limitation in the community edition of jOOQ. Basically jOOQ gets confused if you include a geometry field in a multiset because of the way multisets are implemented under the hood, so you have to explicitly tell it how to render the value in the multiset JSON object. The separate variable is just so we don't have to recalculate that expression every time.